src/components/structures/RoomView.js

```diff
@@ -180,6 +177,11 @@ module.exports = React.createClass({
         roomProm.then((room) => {
             this._calculatePeekRules(room);
             return this._initTimeline(this.props);
+        }).catch(() => {
```

Comment:
Ugh, right. yes.

I need to revisit this code anyway to deal with non-200 responses to the initial event load anyway. In the meantime, could you make this at least log the error or something rather than just swallow it?